src/main/java/movie/storage/model/dto/CinemaHallDto.java

```diff
@@ -0,0 +1,22 @@
+package movie.storage.model.dto;
+
+public class CinemaHallDto {
```

Comment:
we don't use this class at the moment - let's remove it

Reply:
Done